cmd/internal/storage/stdout/stdout.go

```diff
@@ -32,27 +33,62 @@ type stdoutStorage struct {
 }
 
 const (
-	colTimestamp = "timestamp"
-	// CPU Uasge
-	colCpuCumulativeUsage = "cpu_cumulative_usage"
```

Comment:
I'm fine with the rename, but just in-case someone is using these metrics, lets keep the old name around as well, with a link to an issue to remove it in a few releases (v0.39.0)